hw3/templates/index.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1 class="title"> Регистрация </h1>
```

Comment:
После тут нужно обработать flash сообщения. Примерно так
```suggestion
<h1 class="title"> Регистрация </h1>
    {% with messages = get_flashed_messages() %}
	      {% if messages %}
	      <ul>
	          {% for message in messages %}
	          <li>
	                  {{ message }}
	          </li>
	          {% endfor %}
	      </ul>
	      {% endif %}
    {% endwith %}
```